src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -817,6 +849,7 @@ await DismissUIAndRollbackEditsAndEndRenameSessionAsync(
 
     private async Task CommitCoreAsync(IUIThreadOperationContext operationContext, bool previewChanges)
     {
+        CommitStateChange?.Invoke(this, true);
```

Comment:
what is this event for?

Reply:
In legacy UI, when commit starts, we want to disable these buttons, because you can't edit these options once commit starts.
![image](https://github.com/user-attachments/assets/03ddb432-731e-4b24-a701-51cfea26f500)
In new UI, when commit starts this event will make the UI collapse. Because it hides the background indicator tooltip
<img width="433" alt="image" src="https://github.com/user-attachments/assets/e1ced396-0308-43e6-9515-8db80e6330a0">